boards/arm/nrf52833_pca10114/nrf52833_pca10114.dts

```diff
@@ -22,27 +22,27 @@
 	leds {
 		compatible = "gpio-leds";
 		led0_green: led_0 {
-			gpios = <&gpio0 15 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 15 GPIO_INT_LOW_0>;
 			label = "Green LED 0";
 		};
 		led1_red: led_1 {
-			gpios = <&gpio0 20 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 20 GPIO_INT_LOW_0>;
 			label = "Red LED 1";
 		};
 		led1_green: led_2 {
-			gpios = <&gpio0 28 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 28 GPIO_INT_LOW_0>;
```

Comment:
I looked at the Zephyr boards and this is never used to configure things related to interrupts.
I think it should be `GPIO_ACTIVE_LOW` to inform that: `GPIO pin is active (has logical value ‘1’) in low state.`
Same for other places here.

Reply:
> I think it should be `GPIO_ACTIVE_LOW`

Definitely.

Reply:
done.